sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/InsecureHashAlgorithmTest.cs

```diff
@@ -18,6 +18,8 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+#if NETFRAMEWORK // SHA1Managed is sealed in .Net Core and HMACRIPEMD160 is not available
```

Comment:
maybe we need to explore netcore apis? you could leave an FYI note on the appsec channel

Reply:
@eric-therond-sonarsource: what do you think about this? should we take any actions?